algorithms/ar-prfac/java/PrimeFactor.java

```diff
@@ -0,0 +1,20 @@
+public class PrimeFactor{
+
+    public static void primeFactors(int n){
```

Comment:
Codacy found an issue: [Avoid reassigning parameters such as 'n'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6327890)

Reply:
Okay, I will make corresponding changes. 
Do I have to create a new PR?